src/leader/directed_ring/dolev_klawe_rodeh.go

```diff
@@ -0,0 +1,156 @@
+package directed_ring
+
+import (
+	"encoding/json"
+	"fmt"
+	"lib"
+	"log"
+)
+
+type statusDovelKlaweRodeh int
+const (
+	maximum  statusDovelKlaweRodeh = 0
+	active  statusDovelKlaweRodeh = 1
+	passive statusDovelKlaweRodeh = 2
+)
+
+type stateDovelKlaweRodeh struct {
+	MaxV   int
+	LeftV  int
+	Status statusDovelKlaweRodeh
+}
+
+type messageTypeDovelKlaweRodeh int
```

Comment:
Probably my main question for this PR: shouldn't messageTypeDovelKlaweRodeh be more readable as an enum like statusDovelKlaweRodeh? Do they have assigned names in the paper?